RELEASE.md

```diff
@@ -1,39 +1,35 @@
 # Release Process
-zfs-localpv follows a monthly release cadence. The scope of the release is determined by contributor availability. The scope is published in the [Release Tracker Projects](https://github.com/orgs/openebs/projects/10).
+LocalPV ZFS follows or tries to follow semantic versioning principles as specified here https://semver.org. It follows a on quarterly release cadence for minor version releases. The scope of the release is determined by contributor availability. The scope is published in the [Release Tracker Projects](https://github.com/orgs/openebs/projects/78).
```

Comment:
```suggestion

LocalPV ZFS follows or tries to follow semantic versioning principles as specified here https://semver.org. It follows a on quarterly release cadence for minor version releases. The scope of the release is determined by contributor availability. The scope is published in the [Release Tracker Projects](https://github.com/orgs/openebs/projects/78).
```

Reply:
> LocalPV ZFS follows or tries to follow semantic versioning principles as specified here https://semver.org.

Should it be follows?
I'd be more inclined to use "tries to" on the quarterly release cadence

Reply:
Done. Made it tries to follow.